index.html

```diff
@@ -0,0 +1,311 @@
+                                <li class="skill-card-tech-tag rspec"><p class="skill-card-tech-tag-text">RSpec</p></li>
+                            </div>
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag capybara"><p class="skill-card-tech-tag-text">Capybara</p></li>
+                                <li class="skill-card-tech-tag selenium"><p class="skill-card-tech-tag-text">Selenium</p></li>
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+                <!-- skills -->
+                <section class="skill-card">
+                    <!-- icon -->
+                    <div class="skill-card-icon">
+                        <div class="skill-card-icon-container"><img class="skill-card-icon-img" src="img/skill/skills-img.svg" alt=""></div>
+                    </div>
+                    <!-- title and tags -->
+                    <div  class="skill-card-content">
+                        <h3 class="skill-card-title">Skills</h3>
+                        <ul class="skill-card-rows">
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag codekit"><p class="skill-card-tech-tag-text">Codekit</p></li>
+                                <li class="skill-card-tech-tag github"><p class="skill-card-tech-tag-text">GitHub</p></li>
+                                <li class="skill-card-tech-tag codepen"><p class="skill-card-tech-tag-text">Codepen</p></li>
+                                <li class="skill-card-tech-tag gitlab"><p class="skill-card-tech-tag-text">Gitlab</p></li>
+                            </div>
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag terminal"><p class="skill-card-tech-tag-text">Terminal</p></li>
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+            </div>
+        </section>
+        
+        <!-- Form Section -->
+        <section id="form">
+            <!-- text -->
+            <div id="form-text">
+                <p id="form-text-item">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <!-- form -->
+            <form action="https://formspree.io/f/xyyakzgb" method="POST" id="form-content">
+                <input type="text" name="FullName" max-length="30" placeholder="Full Name" class="form-row" id="ignoreindesktop" required>
+                <input type="email" name="email" max-length="30" placeholder="Email Address" id="" class="form-row" required>
+                <textarea name="comment" id="" cols="30" rows="10" max-length="500" placeholder="Typing..." class="form-row textareas"></textarea>
```

Comment:
The form is submitted even when the textarea field is empty. This is not suppose to be so. Kindly add validation `eg: required` to prevent the form from submitting an empty textfield/textarea. Remember, this is part of the project requirements as stated by the previous reviewer. :wink: